dbhydra/src/xlsx_db.py

```diff
@@ -83,9 +84,10 @@ def create_database(self):
             print("Database directory created")
         except FileExistsError:
             print("Database directory already exists")
-            
-            
-    
+
+    @contextlib.contextmanager
+    def transaction(self):
+        yield None
```

Comment:
Give here explanation please - "for compatibility with other dialects supporting transactions" or something like that